addon/xml/helpers.js

```diff
@@ -9,6 +9,10 @@ export function echDate(date) {
     : date;
 }
 
+export function not(a) {
```

Comment:
I think by now there is already a unset function merged maybe this solves your use-case?

Reply:
I used it for the `(not (eq ...` in the address xml template.